src/pages/SettingsPage/PreferencesPage.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import {View} from 'react-native';
+import RNPickerSelect from 'react-native-picker-select';
+import {withOnyx} from 'react-native-onyx';
+import PropTypes from 'prop-types';
+
+import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
+import {redirect} from '../../libs/actions/App';
+import HeaderGap from '../../components/HeaderGap';
+import ROUTES from '../../ROUTES';
+import ONYXKEYS from '../../ONYXKEYS';
+import styles from '../../styles/styles';
+import Text from '../../components/Text';
+import Icon from '../../components/Icon';
+import NameValuePair from '../../libs/actions/NameValuePair';
+import CONST from '../../CONST';
+import {DownArrow} from '../../components/Icon/Expensicons';
+
+
+const propTypes = {
+    // The chat priority mode
+    priorityMode: PropTypes.string,
+};
+
+const defaultProps = {
+    priorityMode: CONST.PRIORITY_MODE.DEFAULT,
+};
+
+const priorityModes = {
+    default: {
```

Comment:
Should we call this `mostRecent` instead of `default`? I think it's more descriptive, and I'd suggest updating the constant from `CONST.PRIORITY_MODE.DEFAULT` to `CONST.PRIORITY_MODE.MOST_RECENT`

Reply:
existing as well, let me know if you still prefer this to be updated

Reply:
@marcaaron thoughts?

Reply:
I think this is out of scope for this PR which is just copying and pasting this over from another place. But if the naming is a concern we can spin up a new issue to look into it?